src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsNotToProceedRespondentNotificationHandler.java

```diff
@@ -70,25 +70,17 @@ public List<CaseEvent> handledEvents() {
     private CallbackResponse notifyRespondentSolicitorForClaimantConfirmsNotToProceed(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
         CaseEvent caseEvent = CaseEvent.valueOf(callbackParams.getRequest().getEventId());
-        String template = null;
         var recipient = isCcNotification(callbackParams)
             ? caseData.getApplicantSolicitor1UserDetails().getEmail()
             : caseData.getRespondentSolicitor1EmailAddress();
 
-        if (SPEC_CLAIM.equals(caseData.getCaseAccessCategory())) {
-            template = isCcNotification(callbackParams)
-                ? notificationsProperties.getClaimantSolicitorConfirmsNotToProceedSpec()
-                : notificationsProperties.getRespondentSolicitorNotifyNotToProceedSpec();
-        } else {
-            template = notificationsProperties.getClaimantSolicitorConfirmsNotToProceed();
-        }
         if (isRespondentSolicitor2Notification(callbackParams)) {
             recipient = caseData.getRespondentSolicitor2EmailAddress();
         }
 
         notificationService.sendMail(
             recipient,
-            template,
+            addTemplate(caseData, callbackParams),
```

Comment:
I think better name for the method is getTemplate rather than addTemplate

Reply:
change the name as `getTemplate`